modules/hcp-ibm-power-create-heterogeneous-nodepools-agent-hc.adoc

```diff
@@ -0,0 +1,201 @@
++
+[source,terminal]
+----
+$ export DB_VOLUME_SIZE="10Gi"
+$ export FS_VOLUME_SIZE="10Gi"
+$ export OCP_VERSION="4.18.0"
+$ export OCP_MAJMIN=${OCP_VERSION%.*}
+$ export ARCH_X86="x86_64"
+$ export OCP_RELEASE_VERSION_X86=$(curl -s https://mirror.openshift.com/pub/openshift-v4/${ARCH_X86}/clients/ocp/${OCP_VERSION}/release.txt | awk '/machine-os / { print $2 }')
+$ export ISO_URL_X86="https://mirror.openshift.com/pub/openshift-v4/${ARCH_X86}/dependencies/rhcos/${OCP_MAJMIN}/${OCP_VERSION}/rhcos-${OCP_VERSION}-${ARCH_X86}-live.${ARCH_X86}.iso"
+$ export ROOT_FS_URL_X8="https://mirror.openshift.com/pub/openshift-v4/${ARCH_X86}/dependencies/rhcos/${OCP_MAJMIN}/${OCP_VERSION}/rhcos-${OCP_VERSION}-${ARCH_X86}-live-rootfs.${ARCH_X86}.img"
+$ export ARCH_PPC64LE="ppc64le"
+$ export OCP_RELEASE_VERSION_PPC64LE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/${ARCH_PPC64LE}/clients/ocp/${OCP_VERSION}/release.txt | awk '/machine-os / { print $2 }')
+$ export ISO_URL_PPC64LE="https://mirror.openshift.com/pub/openshift-v4/${ARCH_PPC64LE}/dependencies/rhcos/${OCP_MAJMIN}/${OCP_VERSION}/rhcos-${OCP_VERSION}-${ARCH_PPC64LE}-live.${ARCH_PPC64LE}.iso"
+$ export ROOT_FS_URL_PPC64LE="https://mirror.openshift.com/pub/openshift-v4/${ARCH_PPC64LE}/dependencies/rhcos/${OCP_MAJMIN}/${OCP_VERSION}/rhcos-${OCP_VERSION}-${ARCH_PPC64LE}-live-rootfs.${ARCH_PPC64LE}.img"
+----
++
+[source,terminal]
+----
+envsubst <<"EOF" | oc apply -f -
+apiVersion: agent-install.openshift.io/v1beta1
+kind: AgentServiceConfig
+metadata:
+ name: agent
+spec:
+  databaseStorage:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: ${DB_VOLUME_SIZE}
+  filesystemStorage:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: ${FS_VOLUME_SIZE}
+  osImages:
+    - openshiftVersion: "${OCP_VERSION}"
+      version: "${OCP_RELEASE_VERSION_X86}"
+      url: "${ISO_URL_X86}"
+      rootFSUrl: "${ROOT_FS_URL_X8}"
+      cpuArchitecture: "${ARCH_X86}"
+    - openshiftVersion: "${OCP_VERSION}"
+      version: "${OCP_RELEASE_VERSION_PPC64LE}"
+      url: "${ISO_URL_PPC64LE}"
+      rootFSUrl: "${ROOT_FS_URL_PPC64LE}"
+      cpuArchitecture: "${ARCH_PPC64LE}"
+EOF
+----
++
+The above configuration allows you to create ISO for both x86_64 and ppc64le architectures.
+
+
+[id="hcp-configure-dns_{context}"]
+== Configure DNS
+
+*.apps.<cluster_name> record can be pointed to either one of the worker nodes where ingress application is hosted, or if you are able to set up a load balancer on top of the worker nodes, point this record to this load balancer. When you are creating a heterogeneous nodepool, please make sure the workers are reachable from each other or keep them in the same network.
+
+.Procedure
+
+. Create a Hosted Cluster
++
+Use a multi-architecture release image when creating the cluster to enable support for heterogeneous nodepools. Find the latest multi arch images from here.
```

Comment:
```suggestion
Use a multi-architecture release image when creating the cluster to enable support for heterogeneous node pools. Find the latest multi-architecture images from here.
```

- "....from here" is a little vague. You can say something like, by using the following command/example/configuration, and so on.